ongr/elasticsearch-bundle/5.0/config/packages/ongr_elasticsearch.yaml

```diff
@@ -0,0 +1,10 @@
+# Read the documentation: http://docs.ongr.io/ElasticsearchBundle
+ongr_elasticsearch:
+    managers:
+        default:
+            index:
+                hosts:
+                    - 127.0.0.1:9200
+                index_name: my-app-index
+            mappings:
+                - AppBundle
```

Comment:
We don"t have `AppBundle` in Flex's project, so that looks wrong to me.

Reply:
I could rename it to the `YourBundleWithDocuments` or smth like this. It's an example of mapping documents from the bundle.

Reply:
Or maybe `AcmeBundle` ?

Reply:
What I mean is that we don't have bundles anymore in a Flex application for project's code, but a simple `App` namespace under `src/`.